packages/integration-karma/test/component/dynamic-imports/index.spec.js

```diff
@@ -0,0 +1,21 @@
+import { createElement } from 'lwc';
+import Container from 'x/dynamic';
+import DynamicCtor from 'x/ctor';
+import { registerForLoad, clearRegister } from 'test-utils';
+
+beforeEach(() => {
+    clearRegister();
+});
+
+it('should call the loader', () => {
+    registerForLoad('x-ctor', DynamicCtor);
+
+    const elm = createElement('x-dynamic', { is: Container });
+    document.body.appendChild(elm);
+
+    return Promise.resolve().then(() => {
+        const span = elm.shadowRoot.querySelector('span');
+        expect(span).not.toBeNull();
```

Comment:
@caridy This test should pass, take a look.

Reply:
you ere searching for the `span` inside `x-dynamic` instead of inside `x-ctor`. I will fix it, and add a lot more tests.